backend/src/main/java/ai/verta/modeldb/configuration/EnableReconcilers.java

```diff
@@ -0,0 +1,16 @@
+package ai.verta.modeldb.configuration;
+
+import org.springframework.context.annotation.Condition;
+import org.springframework.context.annotation.ConditionContext;
+import org.springframework.core.type.AnnotatedTypeMetadata;
+
+public class EnableReconcilers implements Condition {
+  @Override
+  public boolean matches(ConditionContext context, AnnotatedTypeMetadata metadata) {
+    return !liquibaseRunSeparately();
+  }
+
+  boolean liquibaseRunSeparately() {
+    return "true".equalsIgnoreCase("true");
```

Comment:
??? this is not right, obviously. Also, I think we also need to make sure that we don't have liquibase enabled at all, correct?

Reply:
So I have Removed liquibase migration flag logic and just made a reliable in a separate migration flag.

Reply:
@jkwatson-verta it was my problem we have used flag at multiple places so while doing testing i was forget to change flag at other place so i got errors. So your logic is right. i am updating PR.

Reply:
Updated PR.

Reply:
This still has the same issue with the database connectivity not working that I mentioned yesterday. :(

Reply:
the database connectivity not working is a separate issue. No?

Reply:
sure, but this PR leaves the db connectivity broken, so we can't merge it.

Reply:
I have checked at local and it is working fine. i do not face any issue with DB connectivity.

Reply:
that may be so, but this branch does not work when deployed to kubernetes, so we can't merge it.

Reply:
Fix DB connection issue at: https://github.com/VertaAI/modeldb/pull/3046/commits/6a735369a89d1f2e280a84dd04ed0e2d77969669 it was due session factory initialization. before this code we will initialized it from migration code but now it is conditional so when service up it was skip and further health check not getting that. so now it is fixed.

Reply:
After deploying on my dev environment `aj` i have noticed it is works fine as expected.

Reply:
cool. will verify, approve and merge! Thanks for tracking it down.